### PR TITLE
Add ForceDrain description in 2012 Version

### PR DESCRIPTION
**Pull request by velkovb**
make examples formatting uniform in all versions.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @tseward

Peer review needed for this PR. Thanks.

File: docset/winserver2012r2-ps/failoverclusters/Suspend-ClusterNode.md [hunk 1]
```diff
@@ -218,7 +220,7 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/failoverclusters/Suspend-ClusterNode.md]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

**Comment by v-raaba** [comment]
Hi @velkovb  - We understand that these [PR criteria](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=master) are new, and we appreciate your patience with the process. 

**Following are blocking issues that need to be resolved before we can merge:**

Conflicting files
docset/windows/failoverclusters/suspend-clusternode.md 


***

**Reference**

[Criteria used for pull request reviews](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=master)



***

<Use this space to insert any comments.>

Thanks

**Comment by e0i** [comment]
Merge conflicts have been resolved.

**Pull request merged**